mlflow/ml-package-versions.yml

```diff
@@ -75,7 +75,7 @@ tensorflow:
 
   autologging:
     minimum: "1.15.4"
-    maximum: "2.4.1"
+    maximum: "2.5.0"
```

Comment:
Seems that the right version is fetched: https://pypi.org/project/tensorflow/#history